### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by tsohleDev**
# V in MVC

in this PR I:
- Create **action view** for User and Post controllers
- Connect models to views using controllers
- Make data dynamic
- style the action views

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @tsohleDev 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here are key points to consider when documenting your README.

-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✖️ 

For example:
image
image
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/index.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<%= render partial: 'shared/user', collection: @users %>
```
Comment (Owusu-Desmond) [inline app/views/users/index.erb]: - [ ] Please make sure that the User Controller `#index` action renders all users to its corresponding Views index page (`app/views/users/index.html.erb`). Why don't you name your views files as `.html.erb`? Currently have this error while rending your user index page so make sure that you fix this issue :point_down:  and ensure that all your view are rendering correctly. 
  
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @tsohleDev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```
Comment (Whoistolu) [inline README.md:162]: - [ ] [OPTIONAL] In order to have a professionally documented README file, kindly ensure that a URL of the issue page is attracted in this section.

**Review by Whoistolu — APPROVED** [review]
### Hi @tsohleDev 

Great job implementing the recommendation of the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (tsohleDev) [reply README.md:162]: ohh yes thanks

**Pull request merged**